### PR TITLE
Forms project completed

### PR DESCRIPTION
**Pull request by mkezfarias**
Good afternon Mr. TSE,

We have completed the forms rails project and would like to request a review if you may.

Thank you in advanced.

Luis Puente / Carlos de Medeiros

### CODE REVIEW

File: app/views/users/new.html.erb
Comment (Ispirett) [inline app/views/users/new.html.erb:25]: This form is throwing and error.

```
param is missing or the value is empty: user
user_params     params.require(:user).permit(:username,:password,:email)   end    end
-- | --



```

**Review by Ispirett — CHANGES_REQUESTED** [review]
# status changes: required :+1: 
Great work so far @mkezmkez  @camfilho  with this project.
There is one bug to address :bug:  please see below.

File: app/views/users/new.html.erb
Comment (camfilho) [reply app/views/users/new.html.erb:25]: Hello there,

Could you tell us how to replicate the bug?

Comment (Ispirett) [reply app/views/users/new.html.erb:25]: I commented the others forms went to users/new and tried to create a new record .

**Comment by camfilho** [comment]
## For the new reviewer
Please uncomment line 8 on Users controller in order to avoid the above error

File: app/views/users/edit.html.erb
Comment (maelfosso) [inline app/views/users/edit.html.erb]: Here it should be a password field

File: app/controllers/users_controller.rb
Comment (maelfosso) [inline app/controllers/users_controller.rb]: Remove commented lines

File: app/views/users/new.html.erb
Comment (maelfosso) [inline app/views/users/new.html.erb]: For password, you should use a password field

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required

Hello @mkezmkez  @camfilho 

Great work so far @mkezmkez @camfilho with this project.
Just a few things to changes to make it approve

Kindly read the comments below, make the corresponding changes and submit a new code review request.
Thanks.

**Mael FOSSO**
[Github](https://github.com) | [Twitter](https://twitter.com) | [LinkedIn](https://www.linkedin.com/in/mael-elvis-fosso-650b6346/)

File: app/views/users/edit.html.erb
Comment (samgaco) [inline app/views/users/edit.html.erb]: - [ ] Please, pass this into a password field as required by @maelfosso before

**Review by samgaco — APPROVED** [review]
Hi @camfilho and @mkezmkez :high_brightness: 

One requirement from the previous review is still not addressed, please correct it before merging. Since it is just a small fix I will approve the project, so after committing that change you can merge and continue :+1: 


---------------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**
